content/jobstos.md

```diff
@@ -0,0 +1,69 @@
+---
+title: PyBaMM Job Board Terms and Conditions
+shortcutDepth: 1
+summary: "Terms and conditions for posting jobs on the PyBaMM Jobs Board"
+---
+
+## Introduction
+
+These Terms and Conditions govern the job posting services offered by PyBaMM, a project under [NumFOCUS, Inc.](https://numfocus.org). Companies agree to pay a fee for job postings related to battery modeling positions on PyBaMM’s job board. All income generated from this service is considered earned income and is mission-aligned.
+
```

Comment:
We use multiple names for the same page: "Jobs board," "Job Board, " "PyBaMM's job board," "PyBaMM Job Board.". Should we use one everywhere for consistency?

"PyBaMM Jobs Board" sounds good to me, IMO.

Reply:
Sounds good, will do it.